doc/policy/mempool-limits.md

```diff
@@ -0,0 +1,65 @@
+# Mempool Limits
+
+## Definitions
+
+Given any two transactions Tx0 and Tx1 where Tx1 spends the output of Tx0:,
+Tx0 is the *parent* of Tx1 and Tx1 is the *child* of Tx0.
```

Comment:
nit:
```suggestion
Given any two transactions Tx0 and Tx1 where Tx1 spends an output of Tx0,
Tx0 is the *parent* of Tx1 and Tx1 is a *child* of Tx0.
```

Reply:
Good catch, changed both s/the/a

Reply:
This is still a weird `:,` at the end of the first line, apart from that it looks good!